Documentation/ApiOverview/ContentElements/_Migration/_typoscript.diff

```diff
@@ -1,20 +1,24 @@
 -tt_content.list.20.examples_pi1 = USER
 -tt_content.list.20.examples_pi1 {
-+tt_content.examples_pi1 = USER
++tt_content.examples_pi1 =< lib.contentElement
 +tt_content.examples_pi1 {
-     userFunc = MyVendor\Examples\Controller\ExampleController->example
-     settings {
-         singlePid = 42
-         listPid = 55
-     }
-     view {
-         templateRootPath = {$templateRootPath}
-         partialRootPath = {$partialRootPath}
-         layoutRootPath = {$layoutRootPath}
+     20 = USER
+     20 {
+         userFunc = MyVendor\Examples\Controller\ExampleController->example
+         settings {
+             singlePid = 42
+             listPid = 55
+         }
+         view {
+             templateRootPath = {$templateRootPath}
```

Comment:
This is Setup TypoScript. So templateRootPath is templateRootPaths and it's array now. Same for the other both properties. Further we should provide a default (key 0 or 10) and overwrite that value from site settings or constants (10 or 20)

Reply:
Updated it